src/App.jsx

```diff
@@ -1,8 +1,196 @@
 import './App.css';
+import {amountOfTvSold} from './constants/helperSoldTV.js'
+import {amountOfTVBought} from "./constants/amountBought.js";
+import {amountToBeSold} from "./constants/amountToBeSold.js";
+import {allTV, tvName} from "./constants/helperTVString.js";
+import {priceOfTV} from "./constants/tvPrice.js";
+import {tvSize} from "./constants/tvSize.js";
+import {tvImage} from "./constants/tvImage.js";
+import checkImage from './assets/check.png';
+import minusImage from './assets/minus.png';
+import {comparePrice, compareRefresh, compareSoldItems} from "./constants/assets/calculations.js";
+
 
 function App() {
-  return (
-    <h1>Begin hier met met maken van de applicatie!</h1>
+
+    function meestVerkocht() {
+        const sortedItems = [...allTV].sort(compareSoldItems);
+        console.log(sortedItems);
+    }
+
+    function goedkoopste() {
+        const sortedItems = [...allTV].sort(comparePrice);
+        console.log(sortedItems)
+    }
+
+    function sport() {
+        const sortedItems = [...allTV].sort(compareRefresh);
+        console.log(sortedItems);
+    }
+
+    function checkAndMinus(array) {
+        const availableOptions = allTV[array].options.map((option) => {
+            if (option.applicable === true) {
```

Comment:
Dit zou je nog kunnen verkorten
```suggestion
            if (option.applicable) {
```